pkgs/by-name/pa/parca/package.nix

```diff
@@ -0,0 +1,82 @@
+{
+  buildGoModule,
+  faketty,
+  fetchFromGitHub,
+  lib,
+  nodejs,
+  pnpm,
+  stdenv,
+}:
+let
+  version = "0.22.0";
+
+  parca-src = fetchFromGitHub {
+    owner = "parca-dev";
+    repo = "parca";
+    rev = "v${version}";
+    hash = "sha256-iuTlKUmugRum0qZRhuw0FR13iE2qrQegTgwpAvgJSXk=";
+  };
+
+  ui = stdenv.mkDerivation rec {
```

Comment:
I would get rid of rec and use finalAttrs pattern.

Reply:
Fixed! thanks!